lib/db.dart

```diff
@@ -12,8 +17,14 @@ class Db {
   final _log = Logger('Db');
 
   Future<Database> init() async {
-    final path = join(await getDatabasesPath(), dbPath);
-    _log.info('DbPath : $path');
+    var path;
+    if (Platform.isLinux) {
+      sqflite_ffi.sqfliteFfiInit();
+      databaseFactory = sqflite_ffi.databaseFactoryFfi;
+      path = join(dataHome.path, globals.appName, dbPath);
```

Comment:
instead of `globals.appName` I thought about using https://pub.dev/packages/package_info_plus and retrieve the app name using that, but decided against introducing another dependency. Let me know if the dependency would be OK and preferred to the global hack, or if there's a better way.

Reply:
thanks, I indeed prefer to not add a dependency for this. A simple `const` seems ok in this case.